app/js/question_controller.js

```diff
@@ -166,7 +166,8 @@
     evt.preventDefault();
     var comment = question_field.value;
 
-    if (comment === '') {
+    // Disable submit if the comment is blank or with some spaces
+    if (comment === '' || comment.replace(/\s/g, '') === '') {
```

Comment:
We can do a simpler version of this by using [`trim()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)